### PR TITLE
Site is updated with 3.2.2, 4.0.0 release details

### PR DESCRIPTION
**Pull request by solomax**

### CODE REVIEW

File: pelicanconf.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+site:
+  name: Apache Template
```
Comment (ilgrosso) [inline pelicanconf.yaml]: Shouldn't this be "Apache OpenJPA"?

File: content/index.md [hunk 2]
```diff
@@ -12,6 +12,8 @@ Apache OpenJPA is a Java persistence project at [The Apache Software Foundation]
  into any Java EE compliant container and many other lightweight
 frameworks, such as Tomcat and Spring. 
 
+The [4.x releases](downloads.html) is an implementation of the Jakarta Persistence API 3.0 specification is the most current production ready release.
+
 The [3.x releases](downloads.html) targetting the [JSR-338 Java Persistence 2.2](http://jcp.org/en/jsr/detail?id=338) is the most current production ready release.
```
Comment (ilgrosso) [inline content/index.md]: Is 3.x still considered "the most current production ready release", even with 4.0.0 released?

**Review by ilgrosso — CHANGES_REQUESTED** [review]
Thanks @solomax 
Would you mind to share how you managed to build the website locally?

**Comment by solomax** [comment]
> Thanks @solomax Would you mind to share how you managed to build the website locally?

DISCLAIMER: some of the steps were required to avoid pollution of my laptop with various Python files/folders

- create empty folder all files will be stored (for ex. `~/work/site-pelican`)
- `cd ~/work/site-pelican` 
- set up `pyenv`
  - `git clone https://github.com/pyenv/pyenv.git .pyenv`
  -  `cd .pyenv && src/configure && make -C src`
  - `cd ..`
  - create file `pyenv.sh` with following content:
```
export PYENV_ROOT="${HOME}/work/site-pelican/.pyenv"
command -v pyenv >/dev/null || export PATH="$PYENV_ROOT/bin:$PATH"
eval "$(pyenv init -)"
```
  - run `source pyenv.sh`
  - install Python 3.8.10: `pyenv install 3.8.10`
  - `pyenv global 3.8.10`
- download original `https://raw.githubusercontent.com/apache/infrastructure-pelican/master/bin/local-pelican-site.sh`
- I did the following changes:
```
--- local-pelican-site.sh.orig	2024-07-11 17:11:16.887533118 +0700
+++ local-pelican-site.sh.new	2024-07-11 17:08:22.698213396 +0700
@@ -8,10 +8,12 @@
 # will not work under basic Windows.
 
 # github prefix for cloning/updating repos
-GH="https://github.com/apache"
+GH="git@github.com:apache"
 
 # site_build directory path. use a /tmp dir by default
-SB="$HOME/pelican-local"
+#SB="$HOME/pelican-local"
+#SB="$HOME/work/_oss/site-pelican"
+SB="$HOME/work/site-pelican"
 
 # infrastructure-pelican repo
 IP="infrastructure-pelican"
@@ -74,7 +76,7 @@
 else
   echo "Cloning $IP"
   # Sometimes useful to add -b <branch> for buildsite testing
-  git clone $GH/$IP 2>&1
+  git clone $GH/$IP.git 2>&1
 fi
 
 IP="$SB/$IP"
@@ -86,7 +88,7 @@
   # cd $REPO && git pull > /dev/null && cd ..
 else
   echo "Cloning $REPO"
-  git clone $GH/$REPO 2>&1
+  git clone $GH/$REPO.git 2>&1
 fi
 REPO="$SB/$REPO"
 # deploy our pipenv if we haven't already
@@ -96,7 +98,7 @@
 if [ ! -f "Pipfile.lock" ];
 then
   echo "Setting up pipenv..."
-  pipenv --three install -r $IP/requirements.txt > /dev/null 2>&1 || 'echo "pipenv install failed!" && exit -1'
+  pipenv install $(< $IP/requirements.txt) || 'echo "pipenv install failed!" && exit -1'
 
 else
   echo "Pipfile.lock found, assuming pipenv exists."
```
- run `bash local-pelican-site.sh openjpa-site`
- got some errors from `pipenv install`
- modify `infrastructure-pelican/requirements.txt` remove all comments starting with `# `
- Build gfm
  - `cd cmark-gfm-0.28.3.gfm.12/build`
  -  `cmake --install-prefix $PWD/.. ..`
  - `make`
  - `make install`
- `cd ../..`
- cd `openjpa-site`
- `git checkout 4.0-site-update`
- `cd ..`
-  run `bash local-pelican-site.sh openjpa-site`


## All subsequent runs:
 - `cd ~/work/site-pelican`
 - `source pyenv.sh`
 - `bash local-pelican-site.sh openjpa-site`

All changes in openjpa-site will be compiled/loaded automatically

-----------------------------------------------
something like this :))

Sorry for the mess

I'm not really like Python :(

**Comment by ilgrosso** [comment]
Thank you @solomax I'll check your instructions later

File: pelicanconf.yaml (hunk 1 above)
Comment (solomax) [reply pelicanconf.yaml]: fixed!

File: content/index.md (hunk 2 above)
Comment (solomax) [reply content/index.md]: fixed!

**Review by ilgrosso — APPROVED** [review]

**Comment by solomax** [comment]
@ilgrosso I believe `4.0.0` should be added to the branch `builds` before merging this :)

What are the steps? :)

Or maybe you can add it? :)

**Comment by ilgrosso** [comment]
> @ilgrosso I believe `4.0.0` should be added to the branch `builds` before merging this :)

I agree.
 
> What are the steps? :)
> 
> Or maybe you can add it? :)

Let me see if I can find something.

**Comment by ilgrosso** [comment]
@solomax added: https://github.com/apache/openjpa-site/tree/builds/4.0.0

**Pull request merged**